spikeinterface/toolkit/postprocessing/principal_component.py

```diff
@@ -64,6 +64,23 @@ def _set_params(self, n_components=5, mode='by_channel_local',
                       dtype=np.dtype(dtype).str)
         
         return params
+    
+    def _filter_units(self, unit_ids, new_waveforms_folder):
+        # creaste new extension folder
+        new_pca_folder = new_waveforms_folder / self.extension_name
+        new_pca_folder.mkdir()
+        # copy parameter file
+        shutil.copyfile(self.extension_folder / "params.json",
+                        new_pca_folder / "params.json")
+
+        # populate folder
+        pca_files = [f for f in (
+            self.extension_folder).iterdir() if f.suffix == ".npy"]
+        for unit in unit_ids:
+            for pca_file in pca_files:
+                if f"{unit}" in pca_file.name:
+                    shutil.copyfile(
+                        pca_file, new_pca_folder / pca_file.name)
```

Comment:
Are the projection also copied ?

Reply:
yes

Reply:
actually no, now the filtered projections are copied and all models (channel-wise or global are copied too)

Reply:
I wanted to say model